src/app/item-feed/ItemFeed.tsx

```diff
@@ -35,13 +29,7 @@ const Item = memo(function Item({ item, tag }: { item: DimItem; tag: TagValue |
     </ItemPopupTrigger>
   );
   return (
-    <motion.div
-      className={styles.item}
-      initial={{ scale: 0, opacity: 0 }}
-      exit={{ scale: 0, opacity: 0 }}
-      animate={{ scale: 1, opacity: 1 }}
-      transition={spring}
-    >
```

Comment:
I gave up on having items animate in and out in the item feed. It was fun, but it becomes way more obvious when you're scrolling and items are appearing/disappearing constantly.